lib/exporter.rb

```diff
@@ -286,7 +286,7 @@ def self.generate_composite_tiff(_coords, _origin, warpables, id, ordered)
   def self.generate_tiles(key, id)
     key = "AIzaSyAOLUQngEmJv0_zcG1xkGq-CXIPpLQY8iQ" if key == "" # ugh, let's clean this up!
     key ||= "AIzaSyAOLUQngEmJv0_zcG1xkGq-CXIPpLQY8iQ"
-    gdal2tiles = "gdal2tiles.py -k --s_srs EPSG:3857 -t #{id} -g #{key} public/warps/#{id}/#{id}.tif public/tms/#{id}/"
+    gdal2tiles = "gdal2tiles.py -k --s_srs EPSG:3857 -z 10-22 -t #{id} -g #{key} public/warps/#{id}/#{id}.tif public/tms/#{id}/"
```

Comment:
OK but so we shouldn't be manually setting the generated zoom levels. We should know the max zoom or we might try to generate a zillion tiles for a big image. Why are we needing to set this manually anyways? It wasn't generating more than one zoom level i guess?

Reply:
Also s_srs should change to match the t_srs of the gdalwarp command above! 4326?

Reply:
```suggestion
    gdal2tiles = "gdal2tiles.py -k --s_srs EPSG:900913 -t #{id} -g #{key} public/warps/#{id}/#{id}.tif public/tms/#{id}/"
```

We will still need to make the zoom level higher, i think!